assets/section-blog-post.css

```diff
@@ -83,10 +83,8 @@
   margin-top: 3rem;
 }
 
-@media screen and (min-width: 750px) {
-  .article-template .share-button__fallback {
-    width: 60%;
-  }
+.article-template__social-sharing + header, .article-template__social-sharing + .article-template__content {
+  margin-top: 1.5rem;
```

Comment:
In which scenario would the `header` come after the social sharing if you cant reorder the header 🤔

Reply:
There is a header element for the title of the blog post which can be re arranged. ![](https://screenshot.click/21-08-opk1k-t637z.png)